jenkins-cicd-vmss/azuredeploy.parameters.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "jenkinsVMAdminUsername": {
+      "value": "GEN-UNIQUE"
+    },
+    "jenkinsVMAdminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "jenkinsDnsPrefix": {
+      "value": "GEN-UNIQUE"
+    },
+    "repositoryUrl": {
+      "value": "https://github.com/Azure/azure-quickstart-templates"
```

Comment:
remove this and use a defaultValue in the template

Reply:
that's updated